jdaviz/configs/cubeviz/plugins/tests/test_tools.py

```diff
@@ -1,11 +1,10 @@
 import pytest
 import numpy as np
-from echo import delay_callback
+from numpy.testing import assert_allclose
 from regions import RectanglePixelRegion
 
 
-@pytest.mark.filterwarnings('ignore:No observer defined on WCS')
-def test_spectrum_at_spaxel(cubeviz_helper, spectrum1d_cube_with_uncerts):
+def test_spectrum_at_spaxel_no_alt(cubeviz_helper, spectrum1d_cube_with_uncerts):
```

Comment:
Have to add "_no_alt" to the name, otherwise `-k test_spectrum_at_spaxel` selects all the other tests with the same prefix as well.